src/components/common/YourCompany/YourCompany.vue

```diff
@@ -196,7 +196,7 @@
 
     <!-- Name Translation(s) -->
     <div
-      v-if="isAlterationFiling || isCorrectionFiling"
+      v-if="isAlterationFiling || (isCorrectionFiling && !isTypeFirm)"
```

Comment:
This negative logic isn't future-safe since we could add correction filings for another entity type and this statement might be incorrect.

I'm going to change it to positive logic (ie, include the types we want, not exclude the types we don't).